src/librustdoc/html/static/rustdoc.css

```diff
@@ -173,13 +173,10 @@ code, pre, a.test-arrow {
 	border-radius: 3px;
 	padding: 0 0.1em;
 }
-.docblock pre code, .docblock-short pre code, .docblock code.spotlight {
+.docblock pre code, .docblock-short pre code {
```

Comment:
Shouldn't it be:

```suggestion
.docblock pre code, .docblock-short pre code, .docblock code.notable {
```

?

Reply:
See the commit message in 1fe0fe4. The classes appeared to be unused.